dev/php.ini

```diff
@@ -1,17 +1,28 @@
 [PHP]
-; Prod settings -- see ../prod/php.ini
+; Prod settings
 date.timezone = UTC
-zend.assertions	= 0
-upload_max_filesize = 24M
-post_max_size = 32M
 file_uploads = On
-
-; CI settings - see ../ci/php.ini
-; TODO Change memory_limit to prod value.
-memory_limit = 1024M
-max_execution_time = 0
```

Comment:
Is there a reason we're removing this altogether? It may still be useful in local development, especially since xdebug may paused execution causing the time to be long.

Reply:
These are added in the file later.